src/sass/ui/_button.sass

```diff
@@ -40,17 +40,23 @@
 
 =button
   background: transparent
-  border: none
   border-radius: $le-radius-small
+  border: none
   cursor: pointer
-  font-weight: 700
+  font-family: var(--font-family-default)
+  font-size: $le-font-size
+  line-height: $le-font-size
   padding: $le-space-small
   transition: opacity $le-speed-xslow
 
   &:disabled
     cursor: default
     opacity: $le-opacity-disabled
 
+  &:focus
+    box-shadow: 0 0 0 2px var(--color-secondary)
```

Comment:
Shadow should use one of the `$le-shadow-*` from the config to be consistent.

Reply:
Added a new one... note it's not so much a traditional shadow but it's more like an outline.